### DISCUSSION TITLE
mpv-lazy播放个别视频，某些帧之间会出现伪影和横纹

### DISCUSSION THREAD

**Discussion by missile2023** [hooke007/mpv_PlayKit#390] [Q&A 问答] [answered]
mpv播放其他视频没问题，播放俺妹bd原盘时发现这种情况，逐帧步进可以看到运动的2帧之间有时会有1帧这种带伪影和横纹的帧，很影响观感，没开补帧。其他播放器试了potplayer、win自带电影与电视、vlc都没有出现这种情况，
截了一小段：https://mega.nz/file/eghmTDgI#lcdv20qqDP2sDE1eauZ_h3JUU23dJSvPtSs9I5_biIU

https://github.com/hooke007/MPV_lazy/assets/149513093/bdd8c4d0-12de-42a0-ac90-4cf12ed19d2d

![-22_25_29](https://github.com/hooke007/MPV_lazy/assets/149513093/e3e9f894-b3e6-4c1a-bcde-3872e9fff6db)
![-22_25_34](https://github.com/hooke007/MPV_lazy/assets/149513093/f3b03766-42fb-4ac6-a27c-b234d710ad83)

**Comment by hooke007** [comment, marked as answer]
隔行视频，正常现象。手动开启反交错

**Reply by missile2023** [reply]
谢谢！解决了
